buildPermissions.py

```diff
@@ -0,0 +1,252 @@
+        name="Can add departments to the club",
+        content_type=department_type)
+    add_permission(
+        codename="change_department",
+        name="Can change departments at the club",
+        content_type=department_type)
+    add_permission(
+        codename="delete_department",
+        name="Can delete departments from the club",
+        content_type=department_type)
+    add_permission(
+        codename='add_certification',
+        name="Can add gear checkout certifications",
+        content_type=certification_type
+    )
+    add_permission(
+        codename='change_certification',
+        name="Can change gear checkout certifications",
+        content_type=certification_type
+    )
+    add_permission(
+        codename='delete_certification',
+        name="Can delete gear checkout certifications",
+        content_type=certification_type
+    )
+    board.permissions.set(all_permissions)
+    board.save()
+
+
+def build_admin():
+    """Create the admin group, which has all possible permissions"""
+    admin = Group.objects.create(name="Admin")
+    add_permission(
+        codename="add_group",
+        name="Can add permission groups",
+        content_type=group_type
+    )
+    add_permission(
+        codename="change_group",
+        name="Can change permission groups",
+        content_type=group_type
+    )
+    add_permission(
+        codename="delete_group",
+        name="Can delete permission groups",
+        content_type=group_type
+    )
+    add_permission(
+        codename="delete_gear",
+        name="Can permanently delete gear from the database (DANGER)",
+        content_type=gear_type
+    )
+    add_permission(
+        codename="delete_staffer",
+        name="Can delete staffers",
+        content_type=staffer_type
+    )
+    add_permission(
+        codename="add_question",
+        name="Can add quiz questions",
+        content_type=question_type
+    )
+    add_permission(
+        codename="change_question",
+        name="Can change quiz questions",
+        content_type=question_type
+    )
+    add_permission(
+        codename="delete_question",
+        name="Can delete quiz questions",
+        content_type=question_type
+    )
+    add_permission(
+        codename="add_answer",
+        name="Can add quiz answers",
+        content_type=answer_type
+    )
+    add_permission(
+        codename="change_answer",
+        name="Can change quiz answers",
+        content_type=answer_type
+    )
+    add_permission(
+        codename="delete_answer",
+        name="Can delete quiz answers",
+        content_type=answer_type
+    )
+    admin.permissions.set(Permission.objects.all())
+    admin.save()
+    
+    
+def add_permission(codename=None, name=None, content_type=None):
+    """Add the permission, only creating if it does not already exist"""
+    try:
+        with transaction.atomic():  # Resolves a quirk with how django handles the test database
```

Comment:
What's the quirk?

Reply:
When running tests, django doesn't correctly roll back object creation unless atomic() is explicitly called

Reply:
Ok, then we haven't set up the tests correctly. What we should do is create a test db everytime the tests are running and populate it with the same data. This way we make sure we don't slowly increase the db size and eventually get conflicts